Casks/m/macgamestore.rb

```diff
@@ -1,20 +1,26 @@
 cask "macgamestore" do
-  version "4.3.4,6084"
-  sha256 "9f4b86ba8726de7c00fa129392b5b3cfdb0af7b682dfedfa1e0b00e08ea0cf11"
+  version "5.3.0,6127"
+  sha256 "3cc7c56c08a78c998ed86ad6b698df2be75045be54bfd3d7db61959ae4291622"
 
-  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.tgz"
+  url "https://www.macgamestore.com/api_clientapp/clientupdates/public/core6/MacGameStore_#{version.csv.first}_#{version.csv.second}.zip"
   name "MacGameStore"
   desc "Buy, download, and play your games"
   homepage "https://www.macgamestore.com/app/"
 
   livecheck do
-    url "https://www.macgamestore.com/api_clientapp/clientupdates/public/update.xml"
-    regex(%r{/MacGameStore_(\d+(?:\.\d+)+)_(\d+)\.t}i)
-    strategy :sparkle do |item, regex|
-      item.url.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
+    url "https://www.macgamestore.com/api_clientapp/clientupdates/public/update-2021.xml"
+    regex(/MacGameStore[._-]v?(\d+(?:\.\d+)+)[._-](\d+)\.zip/i)
+    strategy :xml do |xml, regex|
+      xml.get_elements("//item[sparkle:isBetaBuild=0]/enclosure").map do |item|
+        match = item.attributes["url"]&.match(regex)
+        next if match.blank?
+
+        "#{match[1]},#{match[2]}"
+      end
```

Comment:
```suggestion
      xml.get_elements("//item").map do |item|
        next if item.elements["sparkle:isBetaBuild"]&.text&.include?("1")

        match = item.elements["enclosure"]&.attributes["url"]&.match(regex)
        next if match.blank?

        "#{match[1]},#{match[2]}"
      end
```

If we only work with the items where `sparkle:isBetaBuild` is 0, livecheck won't surface versions that don't have this element. There may be something to be said for going through all of the items and skipping the ones where `sparkle:isBetaBuild` is 1.

With this approach, we may end up surfacing a beta version in the future if upstream switches to using `sparkle:channel` to mark unstable versions (as the Sparkle documentation suggests) but it's arguably better to surface an unstable version (which we can identify as unstable and update the `livecheck` block) rather than quietly fail to surface stable versions (which we can't identify from livecheck's output).